### ISSUE TITLE
Media module not working in Neos 3.3.7

### ISSUE THREAD

**Issue by DrillSergeant** [neos/neos-development-collection#1947]
I just ran a `composer update` and Neos got updated to 3.3.7
When opening the media-module I get an exception 

```
Warning: md5_file(resource://Neos.Neos/Public/JavaScript/ContentModule-built.js): failed to open stream: &quot;Neos\Flow\ResourceManagement\Streams\StreamWrapperAdapter::stream_open&quot; call failed in /srv/www/jumpboxpeter/httpdocs/Data/Temporary/Development/Cache/Code/Flow_Object_Classes/Neos_Neos_Utility_BackendAssetsUtility.php line 57
```
The file resource://Neos.Neos/Public/JavaScript/ContentModule-built.js does not exist.

### Steps to Reproduce

1. Update Neos to 3.3.7
2. Open the media module

#### Expected behavior

I see the media module

#### Actual behavior

Exception

Happens in Neos 3.3.7

**Comment by aertmann** [comment]
Hi @DrillSergeant, there was an issue with an automated script building that file. https://github.com/neos/neos-development-collection/releases/tag/3.3.8 has been released fixing the issue. Btw. a release hasn't been tested until it's been announced on our channels, but thanks for reporting.

**Comment by gerhard-boden** [comment]
Packagist has already noticed the newest tag, can confirm it's also fixed in `3.2.8`

**Comment by DrillSergeant** [comment]
@aertmann thank you. I got surprised by the update as I did not read any announcement and assumed it is right now in the release process. Just reported it to be sure you are aware of it.
Thanks for fixing, I will test it as soon as the new version is announced.
